src/main/java/io/spine/text/TextFactory.java

```diff
@@ -97,28 +102,28 @@ public static Text createText(String... lines) {
     }
 
     /**
-     * Ensures that lines do not contain {@linkplain #containsSeparator(CharSequence)
-     * line separators}.
+     * Ensures that lines do not contain
+     * {@linkplain #containsSeparator(CharSequence) line separators}.
      *
      * @throws IllegalArgumentException
-     *         if at least one line contains a {@linkplain #containsSeparator(CharSequence)
-     *         line separator}
+     *         if at least one line contains a
+     *         {@linkplain #containsSeparator(CharSequence) line separator}
      */
     public static void checkNoSeparators(Iterable<String> lines) {
         lines.forEach(TextFactory::checkNoSeparator);
     }
 
     /**
-     * Ensures that charter sequence does not contain a {@linkplain #containsSeparator(CharSequence)
-     * line separator}.
+     * Ensures that charter sequence does not contain a
+     * {@linkplain #containsSeparator(CharSequence) line separator}.
      *
      * @throws IllegalArgumentException
-     *         if the sequence contains a {@linkplain #containsSeparator(CharSequence)
-     *         line separator}
+     *         if the sequence contains a
+     *         {@linkplain #containsSeparator(CharSequence) line separator}
      */
-    public static void checkNoSeparator(CharSequence s) {
-        if (containsSeparator(s)) {
-            throw newIllegalArgumentException("The line contains line separator: `%s`.", s);
+    public static void checkNoSeparator(CharSequence line) {
+        if (containsSeparator(line)) {
+            throw newIllegalArgumentException("The line contains line separator: `%s`.", line);
```

Comment:
Right now the error message does not sound erroneous. Let's say why it is illegal to have a line separator.